### PR TITLE
Add JRE 11

### PR DESCRIPTION
**Pull request by jasonthorsness**
On behalf of Maksym - tested with 8.1, older 8.5, newer 8.5
The change to the user script fixes a build blocker encountered on my machine

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.2.11 - 2023-11-03
+ - Java 11
```
Comment (davidgomes) [inline CHANGELOG.md:4]: Please explain this a bit better

File: scripts/init.sh [hunk 2]
```diff
@@ -45,6 +45,40 @@ sdb-admin -y add-leaf --host 127.0.0.1 --port 3307 --password ${INIT_PW}
 sdb-admin -y update-config --all --set-global --key enable_external_functions --value on
 sdb-admin -y update-config --all --set-global --key http_api --value on
 
+isVersionGE()
```
Comment (davidgomes) [inline scripts/init.sh:48]: Please rename to `isEngineVersionGE`

File: scripts/setup-singlestore-user.sh [hunk 3]
```diff
@@ -19,7 +19,7 @@ set -eu
 if [[ "$RESULT_USER" -eq 0 ]]; then
     # make sure there are no files for these users/groups that we are going to change otherwise
     # this is sketchy and we shouldnt proceed
-    user_files=$(find / -path /proc -prune -o -user "${user999}" -print)
+    user_files=$(find / -path /sys -prune -o -path /proc -prune -o -user "${user999}" -print)
```
Comment (davidgomes) [inline scripts/setup-singlestore-user.sh:22]: What are the changes in this file doing? It's unclear to me

**Comment by davidgomes** [comment]
Just a few minor comments, this looks good.

File: scripts/setup-singlestore-user.sh (hunk 3 above)
Comment (jasonthorsness) [reply scripts/setup-singlestore-user.sh:22]: Without this exclusion (similar to /proc) I got a permission-denied error on build:
```
1.294 find: '/sys/devices/virtual/powercap/intel-rapl-mmio': Permission denied
1.294 find: '/sys/devices/virtual/powercap/intel-rapl': Permission denied
```
I don't understand why this happened on my machine and not others, but it's clear that this exclusion is in the same spirit as /proc which was already there (no need to look for user files under /sys as they are not real files)

**Review by davidgomes — APPROVED** [review]

**Pull request merged**